packages/block-library/src/index.js

```diff
@@ -146,6 +148,12 @@ export const registerCoreBlocks = () => {
 		video,
 	].forEach( registerBlock );
 
+	// This line is just for testing
+	registerBlockCollection( 'core', {
+		title: 'Core',
```

Comment:
Shouldn't it be wrapped with the translation function?

Reply:
Yes, but this line is only for testing

Reply:
As this seems ready to land, this test collection should be removed before we merge